Cargo.toml

```diff
@@ -14,8 +14,10 @@ edition = "2018"
 num-bigint = { version = "0.2", default-features = false, features = ["std"] }
 num-complex = "0.2"
 num-traits = "0.2"
-pest = "1.0"
-pest_derive = "1.0"
+pest = "2.1"
+pest_derive = "2.1"
+# Pin pest_generator to version with reproducibility
+pest_generator = "2.1.2"
```

Comment:
Not released yet, hopefully they will right after/if they land https://github.com/pest-parser/pest/pull/442.